packages/devtools/client/pages/modules/payload.vue

```diff
@@ -11,7 +13,7 @@ definePageMeta({
 })
 
 const client = useClient()
-const payload = computed(() => client.value?.nuxt.payload)
+const payload = computed<NuxtPayload>(() => JSON.parse(JSON.stringify(client.value?.nuxt.payload)))
```

Comment:
I think still, I would avoid this cloning, as it

1. duplicate the object, doubles the memory usage (could even crash when payload is too big)
2. doesn't handle the circular references
3. losing the details when serializing a class or special objects
4. it not always reflecting to the real data, sometimes could cause confusing on debugging